main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -125,11 +125,14 @@ object Main {
           }
 
         case Command.Init =>
-          Bootstrap.init(cwd)(System.err) match {
-            case Validation.Success(_) =>
+          Bootstrap.init(cwd)(System.err).toResult match {
+            case Result.Ok((_, Nil)) =>
               System.exit(0)
-            case failure =>
-              failure.errors.map(_.message(formatter)).foreach(println)
+            case Result.Ok((_, failures)) =>
```

Comment:
Matching on `Result.Ok((_, Nil))` and `Result.Ok((_, failures))`, i.e., list emptiness, is slightly awkward here. Should we just ignore the errors if it is `Ok` or should we try to recover from them somehow? Or is the plan that the compiler automatically recovers during each phase when we are done refactoring?